src/components/Activity/NewRequests.js

```diff
@@ -72,22 +83,29 @@ class NewRequests extends Component {
     );
   }
 
+  closeAllExceptCurrent(id) {
+    let temp = this.state.dataSource._dataBlob.s1.slice();
+    temp.map(value => value.close = id !== value._id);
+
+    this.setState({
+      dataSource: this.state.dataSource.cloneWithRows(temp),
+    });
+  }
+
   render() {
     if (this.state.isEmpty)
       return (
         <View style={styles.container}>
           <Image source={require('../../resources/chat_onboarding.png')}/>
           <View style={styles.titleContainer}>
             <Text style={styles.title}>Make a connection!</Text>
-            <Text style={{ color: '#a6aeae', fontSize: 14, }}>
-              You did not connect with anyone yet.
-            </Text>
           </View>
         </View>
       );
     else {
```

Comment:
Sorry I don't understand what you mean. else part is executed when 'isEmpty' state is equal to false